README.md

```diff
@@ -37,4 +37,6 @@ You can edit Breadcrumbs.sublime-settings to specify the following values:
 |------|------|---------|---------|
 |"breadcrumb_length_limit" | Number | 100 | Trim each line to this many characters to form a breadcrumb |
 | "total_breadcrumbs_length_limit" | Number | 200 | Make sure that the total length of the status is no longer than this many characters, by trimming the longest breadcrumbs first |
-| "breadcrumbs_separator" | String | " ■ " | Separate breadcrumbs using this character |
+| "breadcrumbs_separator" | String | " › " | Separate breadcrumbs using this character |
+| "breadcrumbs_regex" | String | "(?P<name>.*)" | Use only the part that matches the "name" group |
+| "breadcrumbs_statusbar" | Boolean | true | Optionally disable hide the breadcrumbs from the statusbar |
```

Comment:
It would be great to expand this doc with useful examples. 
For example, how to remove trailing `{`, or ignore `if`, `while` and other control structures. Things like that.
I can help with writing regexes, but perhaps you already have some examples in mind?